app/components/component-docs/dataDisplay/CoreChip.docs.js

```diff
@@ -1,20 +1,21 @@
 import {
   CoreClasses,
-  CoreH4,
+  CoreH6,
   CoreTypographyBody1,
   CoreChip,
   CoreStack,
   CoreIcon,
   CoreAvatar,
   CoreBox,
   CoreListItem,
-  CorePaper,
-  CoreH6
+  CorePaper
 } from "@wrappid/core";
 
-import CodeImport from "../../CodeImport";
+import CodeBlock from "../../CodeBlock";
 import CodeSample from "../../CodeSample";
-import ComponentProps from "../../ComponentProps";
+import ComponentDocs from "../ComponentDocs";
+const CORE_COMPONENT = CoreChip;
+const CORE_COMPONENT_NAME = (CoreChip?.displayName || CoreChip?.name);
```

Comment:
In next commit,  `CoreChip?.displayName` will be changed to CORE_COMPONENT?.displayName